Cargo.toml

```diff
@@ -31,3 +31,7 @@ members = [
 [profile.release]
 lto = true
 codegen-units = 1
+
+[patch.crates-io]
+arrow = { git = "https://github.com/apache/arrow-rs.git", rev = "731e132489b99cd688f884642cf20de52aed24d0" }
+parquet = { git = "https://github.com/apache/arrow-rs.git", rev = "731e132489b99cd688f884642cf20de52aed24d0" }
```

Comment:
Relies on https://github.com/apache/arrow-rs/commit/e375bba8bcd95f3b96c09a9227f5f196b19dcabd, will remove this once we have arrow 9.0.1 released.